### PR TITLE
Doc: how to edit today's daily note

### PR DESCRIPTION
**Pull request by apraga**
Simple documentation update.

### CODE REVIEW

File: docs/daily-journal.md [hunk 1]
```diff
@@ -43,3 +43,7 @@ Let's unpack this alias:
 * `$ZK_NOTEBOOK_DIR` is set to the absolute path of the current [notebook](notebook.md) when running an alias. Using it allows you to run `zk daily` no matter where you are in the notebook folder hierarchy.
 * We need to use double quotes around `$ZK_NOTEBOOK_DIR`, otherwise it will not be expanded.
 
+If you want to edit today's note, simply use 
+```sh
+$ zk edit journal/daily
```
Comment (tjex) [inline docs/daily-journal.md:48]: Thanks for the PR! As an alias is set (above), the command will actually be `zk daily`.

It's mentioned on line 33, but it helps make it clearer to put it in here again. But yes, it should be `zk daily` not `zk edit journal/daily`.

**Review by tjex — CHANGES_REQUESTED** [review]
See code comments. Thanks ✨

File: docs/daily-journal.md (hunk 1 above)
Comment (apraga) [reply docs/daily-journal.md]: Thanks, I did not know that. PR edited accordingly !

**Review by tjex — APPROVED** [review]
thanks :)

**Pull request merged**